governance.md

```diff
@@ -70,8 +70,10 @@ Duties
 
 Qualifications
 
-* Nominated by community
-* Appointed by Steering Committee
+* Nominated and Appointed by Steering Committee
+  * 2 year terms, term limit of consecutive 4 years (after, individual must take off 1 year)
+  * Terms should be staggered within the TOC and off by 6 months from SC
+  * Calendar year starts with annual Spinnaker Summit 
```

Comment:
What if this were to move quite dramatically, e.g. to the spring?

Reply:
Fair enough - we could just make it easy w/calendar year. You ok w/that?